client/my-sites/stats/activity-log/index.jsx

```diff
@@ -536,24 +527,58 @@ class ActivityLog extends Component {
 
 		const restoreConfirmDialog = requestedRestoreActivity && (
 			<ActivityLogConfirmDialog
-				applySiteOffset={ this.applySiteOffset }
 				key="activity-rewind-dialog"
-				onClose={ this.handleCloseDialog }
-				onConfirm={ this.handleConfirmDialog }
-				timestamp={ requestedRestoreActivity.activityTs }
-			/>
+				confirmTitle={ translate( 'Confirm Rewind' ) }
+				onClose={ this.dismissRestore }
+				onConfirm={ this.confirmRestore }
+				title={ translate( 'Rewind Site' ) }
+			>
+				{ translate(
+					'This is the selected point for your site Rewind. ' +
+						'Are you sure you want to rewind your site back to {{b}}%(time)s{{/b}}?',
+					{
+						args: {
+							time: this.applySiteOffset(
+								moment.utc( requestedRestoreActivity.activityTs )
+							).format( 'LLL' ),
+						},
+						components: { b: <b /> },
+					}
+				) }
+				<div className="activity-log-confirm-dialog__notice">
+					<Gridicon icon={ 'notice' } />
+					<span className="activity-log-confirm-dialog__notice-content">
+						{ translate(
+							'This will remove all content and options created or changed since then.'
+						) }
+					</span>
+				</div>
```

Comment:
Yeah, the notice should be a prop, but instead of passing all the markup, let's pass only the `This will remove all...` text, so if `notice` is passed, render markup with passed text. Otherwise there's too much markup involved in the parent component.

Reply:
moved in 3e5c68d41